crates/examples/Cargo.toml

```
@@ -5,7 +5,7 @@ edition = { workspace = true }
 name = "hotshot-examples"
 readme = "README.md"
 version = { workspace = true }
-rust-version = "1.65.0"
+rust-version = "1.70.0"
```

I think this should actually just be

```suggestion
rust-version = { workspace = true }
```

I don't know if all crates do this, but I think they probably should? cc @QuentinI

---

Most of our crates don't seem to set `rust-version` at all.
I agree, we should probably inherit MSRV for all crates in workspace.

---

I've updated this using `workspace`!